packages/hint-font-relative-units/README.md

```diff
@@ -0,0 +1,66 @@
+# font-relative-units (`@hint/hint-font-relative-units`)
+
+Ensure font styles use relative units instead of pixels.
+
+## Why is this important?
+
+If website stylesheets define certain styles in pixels, they cannot be easily overridden by custom stylesheets and browser plugins. Some people benefit from altering font size, line height, and letter spacing to make text easier to read.
+
+## What does the hint check?
+
+This hint examines `css` and `scss` files and checks that the following properties are not defined in pixels:
+
+- `font-size`
+- `line-height`
+- `letter-spacing`
+
+### Examples that **trigger** the hint
+
+```css
+font-size: 16px;
+line-height: 24px;
+letter-spacing: 2px;
+```
+
+### Examples that **pass** the hint
+
+```css
+font-size: 1.2em;
+line-height: 2em;
+letter-spacing: 0.15em;
+```
+
+## How to use this hint?
+
+To use it you will have to install it via `npm`:
+
+```bash
+npm install @hint/hint-font-relative-units
+```
+
+Note: You can make `npm` install it as a `devDependency` using the `--save-dev`
+parameter, or to install it globally, you can use the `-g` parameter. For
+other options see
+[`npm`'s documentation](https://docs.npmjs.com/cli/install).
+
+And then activate it via the [`.hintrc`][hintrc]
+configuration file:
+
+```json
+{
+    "connector": {...},
+    "formatters": [...],
+    "parsers": [...],
+    "hints": {
+        "font-relative-units": "error"
+    },
+    ...
+}
+```
```

Comment:
**Note to @webhintio/core:**  If we decide to enable this hint in any of the built-in configurations we should update this section and use some of the wording we have in others. Also we should check this text is not in the templates of `create-hint`.